src/packages/@ncigdc/components/Modals/FirstTimeModal.js

```diff
@@ -29,8 +29,9 @@ const FirstTimeModal = () => (
         GDC Data Release Notes
       </ExternalLink>
       {" "}
-      prior to accessing this web site as the Release Notes provide details about data updates, known issues and workarounds.
+      prior to accessing this web site as the Release Notes provide details about data updates, and known issues and workarounds.
```

Comment:
oh I see the extra 'and', didn't read it carefully, copy and pasted it from the ticket, which was a huge text blob. From the diff, looks like the change is adding this sentence in the warning `Data in the GDC is considered provisional as the GDC applies state-of-the art analysis pipelines which evolve over time.`